### ISSUE TITLE
Cancelling the reload prevents you from shooting until the reload duration is finished

### ISSUE THREAD

**Issue by artemisexe** [HugoZink/IreNFist#85]
Particularly by sprinting. I haven't tried canceling using other methods

**Comment by HugoZink** [comment]
Is this only since the latest update?

This is almost certainly a mod compatibility issue (another mod overriding InF's behavior), but that's the second one so far which is worrying. I might have to put back the mod priority and manually implement Autofire Soundfix.

**Comment by artemisexe** [comment]
Before the latest update actually, which was several minutes ago

**Comment by artemisexe** [comment]
https://youtu.be/BFadN9nvndA here's a quick preview

**Comment by HugoZink** [comment]
To clarify, I pushed an update last night (which is about 12 hours ago). Did this issue happen to you too, say, 24 hours ago? The update I pushed "just now" was meant to fix another issue.

I may have to put the priority back at -1, but not until I get autofire soundfix working alongside it again. If I put the priority back right now, full-auto weapons would be totally silent.

**Comment by artemisexe** [comment]
Yeah, it's likely. 
Also, yesterday (and coincidentally), I noticed that the AMR-16 was silent, or at least muffled; the sound was similar to NPC's CAR4s, but this only happened without Hawks_soundpack mod. I didn't report it because it may be just from the soundpack itself.

**Comment by HugoZink** [comment]
Hawk's soundpack has some weird issues in it (I personally blame PD2's package loader sometimes rejecting the soundbanks for literally no reason), so that's probably more likely to be the reason.

If you use the RPK there's like a 50/50 chance it'll just be the shitty default AK sound too.

Either way, I've decided to put back the mod.txt priority and manually work around AFSF2 not working.

**Comment by HugoZink** [comment]
Fixed.
